e2e/wdio/platform/tests/icon-tab-bar.e2e-spec.ts

```diff
@@ -0,0 +1,171 @@
+            checkSelectingTabs(columnsExample);
+        });
+
+    });
+
+    describe('Process example', () => {
+        it('should check selecting tabs in process example', () => {
+            checkSelectingTabs(processExample);
+        });
+
+        it('should check procces example', () => {
+            expect(isElementDisplayed(processExample + processIcon)).toBe(true, 'process icon is not displayed');
+        });
+
+        it('should check quantity of communicators in process example', () => {
+            const itemsQuantity = getElementArrayLength(processExample + tabBarItem);
+            expect(getElementArrayLength(processExample + processIcon)).toBe(itemsQuantity - 1, 'wrong quantity of process icons and tabs');
+        });
+    });
+
+    describe('iconName only example', () => {
+        it('should check selecting tabs in iconName only example', () => {
+            checkSelectingTabs(iconOnlyExample);
+        });
+
+        // skipped due to https://github.com/SAP/fundamental-ngx/issues/6714
+        xit('should check that no labels in only iconName example', () => {
+            expect(doesItExist(iconOnlyExample + label)).toBe(false, 'label exists but should not');
+            expect(doesItExist(iconOnlyExample + counter)).toBe(false, 'counter exists but should not');
+        });
+    });
+
+    describe('Nested tabs example', () => {
+        it('should check selecting tabs in nested example', () => {
+            checkSelectingTabs(nestedTabsExample);
+        });
+
+        // skipped due to https://github.com/SAP/fundamental-ngx/issues/6743
+        xit('should check tab with expanded list', () => {
+            click(nestedTabsExample + tabBarItem, 3);
+            expect(isElementDisplayed(expandedList)).toBe(true, 'expanded list is not displayed');
+            const itemText = getText(listItem, 1);
+            click(listItem, 1);
+            expect(getText(nestedTabsExample + tabBarItem)).toEqual(itemText, 'text is not changed according to selected item');
+            expect(getAttributeByName(nestedTabsExample + tabBarTab, 'aria-selected', 3)).toBe('true', 'tab is not selected');
+        });
+
+    });
+
+    describe('Reordering example', () => {
+        it('should check selecting tabs in columns example', () => {
+            checkSelectingTabs(reorderingExample);
+        });
+    });
+
+    describe('Overflowing example', () => {
+        it('should check selecting tabs in overflowing example', () => {
+            refreshPage();
+            checkSelectingTabs(overflowingExample);
+        });
+
+        it('should check overflowing', () => {
+            click(overflowButton);
+            expect(isElementDisplayed(expandedList)).toBe(true, 'expanded list is not displayed');
+            const iconName = getElementClass(listItem + icon);
+            const quantityOfDisplayedTabs = getElementArrayLength(overflowingExample + tabBarItem + '[aria-hidden=false]');
```

Comment:
not sure aria-hidden is a reliable attribute to use in the selector.. anything else you could use?